tests/lammps/test_base.py

```diff
@@ -506,7 +506,7 @@ def test_vcsgc_input(self):
         self.job_vcsgc_input.calc_vcsgc(**args)
         self.assertEqual(self.job_vcsgc_input.input.control['fix___vcsgc'], input_string)
 
-        args['temperature_mc'] = 100.,
+        args['temperature_mc'] = 100.
```

Comment:
@liamhuber This broke the tests.  I don't think this is intentional or is it and I'm missing it?

Reply:
Just a copy-paste error from a function signature I bet.